### ISSUE TITLE
Full heal button doesn't look like a button

### ISSUE THREAD

**Issue by ava5627** [DerDiemen/eclipsephase#64]
The full heal/repair button is only text that is underlined on hover that makes it difficult to realize it is a button, especially because it looks so different to the take damage and roll heal buttons
![image](https://github.com/DerDiemen/eclipsephase/assets/40039652/7510e6ec-a2d5-4d5f-9afb-b497a2ddfc12)

**Comment by DerDiemen** [comment]
That‘s true. It‘s actually a text link since it‘s more of a vonveniece feature and can be easily avoided at all by just setting the values manually. 

I wanted to have a clear „secondary“ or even „tertiary“ button in this case.
